### PR TITLE
Upgrade package to support Symfony ^5.0

### PR DESCRIPTION
**Pull request by eli-l**
* update Configuration Tree Builder
* update Commands
* update Tests
* drop older version support due to incompatibility

Consider releasing as version "5.0".

### CODE REVIEW

File: Services/CrontabInstaller.php [hunk 1]
```diff
@@ -66,23 +68,24 @@ public function __construct(CrontabExecutor $executor, LoggerInterface $logger,
...
-    public function install()
+    public function install(): int
     {
         $changed = $this->executor->dump();
 
         if ($changed) {
-            $this->executeInstall();
+            return $this->executeInstall();
         }
+        return Command::FAILURE;
```
Comment (krasilnikovm) [inline Services/CrontabInstaller.php:80]: Are you sure that should return fail status? from my perspective, if config not changed it's not an error

File: composer.json [hunk 2]
```diff
@@ -23,13 +23,15 @@
...
-        "dragonmantank/cron-expression": "^2.0"
+        "symfony/framework-bundle": "^5.0",
+        "symfony/validator": "^5.0",
+        "symfony/console": "^5.0",
+        "dragonmantank/cron-expression": "^3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "5.*"
+        "phpunit/phpunit": "5.*",
+        "symfony/browser-kit": "^5.1",
```
Comment (krasilnikovm) [inline composer.json:33]: why did you add "symfony/browser-kit" and "symfony/css-selector"?  looks like it's not used and can be removed

File: Services/CrontabInstaller.php [hunk 3]
```diff
@@ -11,6 +12,7 @@
 namespace Ecentria\Bundle\CrontabBundle\Services;
 
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
```
Comment (ievaxt) [inline Services/CrontabInstaller.php:15]: This dependency has been added just to use it's constants. Consider replacing this usage by creating private constants in the service class.

File: Services/CrontabInstaller.php (hunk 1 above)
Comment (eli-l) [reply Services/CrontabInstaller.php:80]: _dump_ before _install_ (w/o config changes in between) prevents crontabs from being installed (actually added to OS crontab). For me such behavior is definitely a failure of "**installation**".

File: Services/CrontabInstaller.php (hunk 3 above)
Comment (eli-l) [reply Services/CrontabInstaller.php:15]: I don't think we have a good reason to redefine the same constants here. symfony/console is still direct dependency for the lib and provides runtime environment for the specific script.

File: composer.json (hunk 2 above)
Comment (eli-l) [reply composer.json]: good point, fixed
